tests/phpunit/tests/test-wpml-cornerstone-utils.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use WPML\PB\Cornerstone\Utils;
+
+/**
+ * Class Test_WPML_Cornerstone_Utils
+ *
+ * @group cornerstone
+ */
+class Test_WPML_Cornerstone_Utils extends OTGS_TestCase {
+
+	/**
+	 * @test
+	 */
+	public function test_get_node_id() {
+		$data = [ 'test' => 'value' ];
+		$expected = md5( serialize( $data ) );
+
+		$this->assertEquals( $expected, Utils::getNodeId( $data ) );
+	}
+
+	/**
+	 * @test
+	 * @dataProvider dp_type_is_layout
+	 */
+	public function test_type_is_layout( $type, $expected ) {
+		$this->assertEquals( $expected, Utils::typeIsLayout( $type ) );
+	}
+
+	/**
+	 * Data provider for test_type_is_layout.
+	 */
+	public function dp_type_is_layout() {
+		return [
```

Comment:
@dgwatkins I think you should test all the types that are layouts because that seems important.
eg. all these: [ 'bar', 'container', 'section', 'row', 'column', 'layout-row', 'layout-column', 'layout-grid', 'layout-cell' ]

Reply:
@brucepearson Added, thanks